microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
 keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides
 ---
 
-# Microsoft Edge User Agent String (Desktop)  
+# Retrieving User Agent Data in Microsoft Edge 

```

Comment:
Can we give this a more generic heading to reflect the intended usage?

e.g. something like, "Guidance for web sites to detect Microsoft Edge"

And then in the text just below, lead with a high-level overview of why it's not typically recommended to detect the UA and to instead do capability detection. *Then* we can get into available APIs.

Reply:
Updated to "How Websites can detect MS Edge" & overview added.